plugins/com.google.cloud.tools.eclipse.appengine.ui/META-INF/MANIFEST.MF

```diff
@@ -23,6 +23,7 @@ Import-Package: com.google.cloud.tools.eclipse.appengine.libraries.model,
  org.eclipse.jface.wizard,
  org.eclipse.osgi.util;version="1.1.0",
  org.eclipse.swt,
+ org.eclipse.swt.browser,
```

Comment:
Don't need the SWT Browser!  The simplest is to use SWT's `Program.launch()`.  The better is to use `…getWorkbench().getBrowserSupport().getExternalBrowser().openURL(url)`. The latter uses the user's configured browser.